doc/api/magic-plot.md

```diff
@@ -126,37 +126,37 @@ Shortcut: `%sqlplot hist`
 
 +++
 
-Histogram does not support NULL values, so let's remove them:
+Histogram supports NULL values by skipping them. Now we can
+generate histograms without explicitly removing NULL entries.
 
 ```{code-cell} ipython3
-%%sql --save no_nulls --no-execute
+%%sql --save data_with_nulls --no-execute
 SELECT *
 FROM penguins.csv
-WHERE body_mass_g IS NOT NULL
 ```
 
 ```{code-cell} ipython3
-%sqlplot histogram --table no_nulls --column body_mass_g --with no_nulls
+%sqlplot histogram --table data_with_nulls --column body_mass_g --with data_with_nulls
```

Comment:
We are deprecating the `--with` parameter. Please remove it and verify that results look ok